src/test/groovy/com/google/protobuf/gradle/ProtobufJavaPluginTest.groovy

```diff
@@ -200,6 +200,62 @@ class ProtobufJavaPluginTest extends Specification {
     gradleVersion << GRADLE_VERSIONS
   }
 
+  @Unroll
+  void "testProjectJavaLibrary should be successfully executed (java-only as a library) [gradle #gradleVersion]"() {
+    given: "project from testProjectJavaLibrary"
+    File projectDir = ProtobufPluginTestHelper.projectBuilder('testProjectJavaLibrary')
+            .copyDirs('testProjectBase', 'testProjectJavaLibrary')
+            .build()
+
+    when: "build is invoked"
+    BuildResult result = GradleRunner.create()
+            .withProjectDir(projectDir)
+            .withArguments('build', '--stacktrace')
+            .withGradleVersion(gradleVersion)
+            .forwardStdOutput(new OutputStreamWriter(System.out))
+            .forwardStdError(new OutputStreamWriter(System.err))
+            .withDebug(true)
```

Comment:
Oh, I thought we had deleted all instances of withDebug. We probably want to do that in the future. Don't change anything in this PR, though.